config.rb

```diff
@@ -10,7 +10,7 @@
 activate :i18n
 
 # Layouts
-# https://middlemanapp.com/basics/layouts/
+page "/index.html", :layout => "landing"
```

Comment:
You could use a more straight syntax here:

```suggestion
page "/index.html", layout: "landing"
```